docs/Drawing.md

```diff
@@ -19,4 +19,37 @@ using (var image = new MagickImage(new MagickColor("#ff00ff"), 512, 128))
       .Ellipse(256, 96, 192, 8, 0, 360)
       .Draw(image);
 }
-```
\ No newline at end of file
+```
+
+## Adding Text To Existing Image
+
+![Example of adding text to existing image](./img/example_addTextToExistingImage.jpg)
+
+```C#
+var pathToBackgroundImage = "path/to/background.png";
+var pathToNewImage = "path/to/newImage.png";
+var textToWrite = "Insert This Text Into Image";
+
+/* These settings will create a new caption
```

Comment:
Could you change the `/*` to `//` for each line to make sure it is in the rest of the style of the repo?

Reply:
👍